### ISSUE TITLE
Control+home/end not working as expected in the editor

### ISSUE THREAD

**Issue by jryans** [vector-im/riot-web#12438]
I get bitten on a daily basis by control+home/control+end not working as expected in the editor (i.e. scrolling instead of jumping to start/end of editor message). I'm a screen reader user (and thus keyboard only user), and while this isn't screen reader specific, I wonder if it impacts keyboard only users more; i.e. perhaps most users just use the mouse to jump to the start/end of multi-line messages.

_Originally posted by @jcsteh in https://github.com/vector-im/riot-web/issues/4736#issuecomment-585014832_

**Comment by bwindels** [comment]
Can repro indeed, should be easy to do in `BasicMessageComposer._onKeyDown`

**Comment by jcsteh** [comment]
This is great - thanks! - except for one thing: shift+control+home and shift+control+end (select to start/end) now get treated as control+home/control+end. I guess the code doesn't check for the shift key? Anyway, that's a bit problematic if, for example, you type a message, start editing it, then realise you wrote a lot of garbage after a certain point (not enough coffee or something like that) and want to delete it.

**Comment by jryans** [comment]
Aha, thanks for pointing that out @jcsteh! I have opened https://github.com/vector-im/riot-web/issues/12549 to fix.
